lib/carlo.js

```diff
@@ -27,7 +27,14 @@ const fs = require('fs');
 const util = require('util');
 const {URL} = require('url');
 const EventEmitter = require('events');
-const fsReadFile = util.promisify(fs.readFile);
+const readfile = util.promisify(fs.readFile);
+const fsReadFile = async filename => {
```

Comment:
Why is this one needed? You seem to be catching and re-throwing exception right away.

Reply:
just because there is an 'unhandled rejection'... I can drop it if you think it's unnecessary.

Reply:
Yeah, I think it does not change anything, so we should drop it.

Reply:
Done.